src/PriceCalculatorKata/Product.cs

```diff
@@ -12,6 +12,11 @@ enum AdditionalCostType
         Absolute,
         PriceRelative,
     }
+    enum DiscountType
```

Comment:
I see multiple enums in the same class. This file Product.cs should be dedicated to Product class only. Better move out enums to separate files. Have an Enums folder for enums, and Models folder for models like Product.cs.